gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/process/computer/ranking/pagerank/PageRankVertexProgramTest.java

```diff
@@ -38,7 +38,7 @@
     @LoadGraphWith(MODERN)
     public void shouldExecutePageRank() throws Exception {
         if (graphProvider.getGraphComputer(graph).features().supportsResultGraphPersistCombination(GraphComputer.ResultGraph.NEW, GraphComputer.Persist.VERTEX_PROPERTIES)) {
-            final ComputerResult result = graph.compute(graphProvider.getGraphComputer(graph).getClass()).program(PageRankVertexProgram.build().create(graph)).submit().get();
+            final ComputerResult result = graph.compute(graphProvider.getGraphComputer(graph).getClass()).program(PageRankVertexProgram.build().iterations(30).create(graph)).submit().get();
```

Comment:
What about a test for convergence-based execution?

Reply:
There are plenty of `pageRank()`-tests with some testing convergence and some testing `times()`. See `PageRankTest`. What other test do you have in mind?

Reply:
I was thinking more about a test, that verifies, that there were X iterations (where X is a pre-determined number). Now, with the `maxIterations` concept, it would make even more sense to verify that the PR VP terminates early.

Reply:
Pushed. Added a test for iterations breaking, epsilon breaking, and energy conservation.